src/player.h

```diff
@@ -1155,6 +1155,8 @@ class Player final : public Creature, public Cylinder
 	private:
 		std::forward_list<Condition*> getMuteConditions() const;
 
+		void checkInvalidStair(const Tile* newTile, const Position& newPos, const Position& oldPos);
```

Comment:
I would call it `checkStairs` instead, because 1) you check all of them, not only the invalid 2) "stair" is a single step, "stairs" is the correct term